packages/bitcore-node/migrations/fixUnspentInputs.js

```diff
@@ -40,7 +42,9 @@ class Migration {
 
   processArgs(argv) {
     let defaults = {
-      dryrun: true
+      dryrun: true,
+      chain: 'BTC',
+      network: 'testnet'
```

Comment:
If I run this with `--help`, it automatically starts it with BTC:testnet. I actually think it would be better if we don't have defaults for `chain` and `network`. In a migration script, forcing the user to be specific isn't a bad thing.

Reply:
Good point. I updated it to have no defaults for chain and network.